src/Miningcore/config.json

```diff
@@ -61,7 +61,11 @@
                 }
             ],
             "ipWhitelist": []
-        }
+        },
+        "oidcValidateAudience":  false,
+        "oidcValidAudience": "",
+        "oidcValidIssuer": "https://login-int.norton.com/sso/oidc1/token",
```

Comment:
NSL for now, but I think from the functions we will be able to get a token using the managed identity